### PR TITLE
Access starred questions

### PR DESCRIPTION
**Pull request by Bruce-Connor**
List starred questions with `sx-tab-starred`.
Star and unstar questions with `*` and `C-u *`.

### CODE REVIEW

**Comment by vermiculus** [comment]
271 changed files!? There must have been a merge error somewhere.

File: sx-tab.el [hunk 1]
```diff
@@ -232,6 +232,24 @@ If SITE is nil, use `sx-default-site'."
...
+      :auth t
+      :submethod 'favorites
+      :filter sx-browse-filter)))
+;;;###autoload
+(autoload 'sx-tab-featured
+  (expand-file-name
+   "sx-tab"
+   (when load-file-name
+     (file-name-directory load-file-name)))
+  nil t)
```
Comment (vermiculus) [inline sx-tab.el]: Can we make this autoload business a macro? Is that possible with how `autoload` works?

**Comment by vermiculus** [comment]
:+1:

File: sx-tab.el (hunk 1 above)
Comment (Bruce-Connor) [inline sx-tab.el]: It is now. But it wasn't in Emacs 24.2.
So no. :-(

Comment (Bruce-Connor) [inline sx-tab.el]: Now-a-days `make-autoloads` fully expands macros it runs into, and it recursively autoloads the contents of a `progn`. So we could just `;;;###autoload` the calls to `sx-tab--define`.

The way it was before, `progn` wasn't understood, so there's not much we can do I think. Maybe it still expands macros, might be worth checking.

**Pull request merged**